datadog-agent.yaml

```diff
@@ -367,6 +350,9 @@ subpackages:
                 done
               done
 
+              mkdir -p ${{targets.contextdir}}/usr/lib/python${{vars.py-version}}
+              cp -r /usr/lib/python${{vars.py-version}}/site-packages ${{targets.contextdir}}/usr/lib/python${{vars.py-version}}/
```

Comment:
We should be able to avoid these lines, I think. I see other packages doing `pip install --prefix=/usr --root=${{targets.contextdir}}` which I think would solve it.